### ISSUE TITLE
Getting mean taxa abundances & sd (incl NAs) in phyloseq

### ISSUE THREAD

**Issue by djsturm** [joey711/phyloseq#1543]
Hi all,

I am trying to do what seems to be really simple but somehow I can't work it out nor find the solution online.
I have a phyloseq object (ps) and would like to get the mean abundances of taxa at the Phylum level including their standart deviation **BUT want to keep my NAs in the final output.** 
I think I've been able to do the first part (get mere relative abundances for each sample) using this code:

ps.taxa <- tax_glom(ps, taxrank = "Phylum")
ps.taxa <- transform_sample_counts(ps.taxa , function(x) x/sum(x))
ps.dat <- data.frame(otu_table(ps.taxa))
names(ps.dat) <- ((ps.taxa@tax_table@.Data) [, "Phylum"])

However, this gets rid of the unassigned taxa at Phylum level (i.e. NAs), and I would like to keep those in as a seperate categorie for a number of reasons.

So basically what I want to do is get the associated abundance numbers of the output of plot_composition - which shows me bars for all phyla including taxa that have not been assigned at Phylum level. 

I hope that makes sense. 
Thanks in advance!
